test/optional-test.js

```diff
@@ -273,13 +273,11 @@ describe('Optional.js', function () {
         });
 
         it('.orElseThrow() on empty Optional, throws exception returned by provided function', function () {
-            var expectedErrorMessage = 'an error to throw';
-
             assert.throws(function () {
                 emptyOptional.orElseThrow(function () {
-                    return new Error(expectedErrorMessage);
+                    return new Error('an error to throw');
                 });
-            }, expectedErrorMessage);
```

Comment:
Interesting - this works fine in Node 8, but I shouldn't be passing a string here. Looks like the behaviour of `assert.throws` changed in Node 10.